proto/buf/alpha/registry/v1alpha1/plugin_curation.proto

```diff
@@ -218,6 +218,13 @@ message GetLatestCuratedPluginRequest {
 }
 
 message GetLatestCuratedPluginResponse {
-  // The latest plugin.
   CuratedPlugin plugin = 1;
+  // versions is a semver-sorted list in ascending order.
```

Comment:
I suggest showing latest first, when checking versions we usually see them from the most recent to the oldest. Same for the revisions.

Reply:
Fair, updated.